HW_4/task_2.py

```diff
@@ -0,0 +1,8 @@
+# Напишите функцию для транспонирования матрицы
+
+def transpose(matrix):
+    return [list(row) for row in zip(*matrix)]
```

Comment:
Правильный подход через использование функции `zip`, но можно записать чуть короче.
```suggestion
    return [*map(list, zip(*matrix))]
```